packages/contentful/src/crn/queries/analytics.queries.ts

```diff
@@ -157,3 +157,75 @@ export const FETCH_TEAM_PRODUCTIVITY = gql`
     }
   }
 `;
+
+export const FETCH_USER_COPRODUCTION = gql`
+  query FetchUserCoproduction($limit: Int, $skip: Int) {
+    usersCollection(order: firstName_ASC, limit: $limit, skip: $skip) {
+      total
+      items {
+        sys {
+          id
+        }
+        firstName
+        lastName
+        nickname
+        alumniSinceDate
+        labsCollection(limit: 3) {
+          items {
+            sys {
+              id
+            }
+          }
+        }
+        teamsCollection(limit: 3) {
+          items {
+            team {
+              sys {
+                id
+              }
+              displayName
+              inactiveSince
+            }
+            role
+            inactiveSinceDate
+          }
+        }
+        linkedFrom {
+          researchOutputsCollection(limit: 20) {
+            items {
+              addedDate
+              sharingStatus
+              authorsCollection(
+                limit: 10
+                where: { sys: { id_exists: true } }
```

Comment:
I know I suggested this, but I think we'll have to remove this

```
where: { sys: { id_exists: true } }
```
for now because it increased complexity 😢 
![query-complexity](https://github.com/yldio/asap-hub/assets/16595804/4868e442-45d1-4b1d-a93e-1f2ed2fc14aa)